superset-frontend/src/features/alerts/components/NotificationMethod.tsx

```diff
@@ -178,57 +256,134 @@ export const NotificationMethod: FunctionComponent<NotificationMethodProps> = ({
+                <div className="input-container">
+                  <textarea
+                    name="BCC"
+                    data-test="bcc"
+                    value={bccValue}
+                    onChange={onBccChange}
+                  />
+                </div>
+                <div className="input-container">
+                  <div className="helper">
+                    {t('Recipients are separated by "," or ";"')}
                   </div>
-                  {error && (
-                    <div
-                      style={{
-                        color: theme.colors.error.base,
-                        fontSize: theme.gridUnit * 3,
-                      }}
-                    >
-                      {TRANSLATIONS.EMAIL_SUBJECT_ERROR_TEXT}
-                    </div>
-                  )}
-                </>
-              ) : null}
-            </StyledInputContainer>
-          </div>
-          <div className="inline-container">
-            <StyledInputContainer>
-              <div className="control-label">
-                {t('%s recipients', method)}
-                <span className="required">*</span>
-              </div>
-              <div className="input-container">
-                <textarea
-                  name="recipients"
-                  data-test="recipients"
-                  value={recipientValue}
-                  onChange={onRecipientsChange}
-                />
+                </div>
+              </>
+            )}
+            {/* New buttons container */}
+            <div className="ghost-button">
+              <span
+                className="ghost-button"
+                role="button"
+                tabIndex={0}
+                onClick={() => setCcVisible(true)}
+                style={{ display: ccVisible ? 'none' : 'inline-flex' }}
+              >
+                <Icons.Email className="icon" />
+                {t('Add CC Recipients')}
+              </span>
+              <span
+                className="ghost-button"
+                role="button"
+                tabIndex={0}
+                onClick={() => setBccVisible(true)}
+                style={{ display: bccVisible ? 'none' : 'inline-flex' }}
+              >
+                <Icons.Email className="icon" />
+                {t('Add BCC Recipients')}
+              </span>
+            </div>
+            <div className="control-label">
+              {TRANSLATIONS.EMAIL_SUBJECT_NAME}
+            </div>
+            <div className={`input-container ${error ? 'error' : ''}`}>
+              <input
+                type="text"
+                name="email_subject"
+                value={email_subject}
+                placeholder={defaultSubject}
+                onChange={onSubjectChange}
+              />
+            </div>
+            {error && (
+              <div
+                style={{
+                  color: theme.colors.error.base,
+                  fontSize: theme.gridUnit * 3,
```

Comment:
We’ve got specific theme variables for font sizes - check out typography part of superset theme

Reply:
@kgabryje Could you please review and resolve the conversation?